app/src/main/java/org/mozilla/fenix/tabstray/viewholders/NormalBrowserPageViewHolder.kt

```diff
@@ -106,23 +100,6 @@ class NormalBrowserPageViewHolder(
                 }
             }
         }
-
-        if (focusGroupTabId.isNullOrEmpty()) {
-            // Updates tabs into the normal browser tabs adapter.
-            browserAdapter.observeFirstInsert {
-                val activeTabsList = browserStore.state.getNormalTrayTabs(inactiveTabsAreEnabled)
-                activeTabsList.forEachIndexed { tabIndex, trayTab ->
-                    if (trayTab.id == selectedTab.id) {
-                        // Index is based on tabs above (inactive) with our calculated index.
-                        val indexToScrollTo = inactiveTabAdapter.itemCount + tabIndex
-
-                        containerView.post { layoutManager.scrollToPosition(indexToScrollTo) }
-
-                        return@observeFirstInsert
-                    }
-                }
-            }
```

Comment:
Realized while addressing @rocketsroger's feedback that this should be added back in, as it controls the scroll-to logic for active tabs. Moved it to an `else` after the selected tab id logic block for inactive tabs.